src/GraphQL/ValueConverter.cs

```diff
@@ -33,6 +33,9 @@ static ValueConverter()
             Register(typeof(long), typeof(int), LongToInt);
 
             Register(typeof(double), typeof(decimal), DoubleToDecimal);
+
+            Register(typeof(string), typeof(Uri), ParseUri);
+            Register(typeof(Uri), typeof(Uri), ParseUri);
```

Comment:
Don’t need Uri->Uri registration.  The code checks to see if the value is of the requested type and returns if so.

Reply:
fix'd